crates/testing/tests/tests_1/quorum_proposal_recv_task.rs

```diff
@@ -95,6 +94,7 @@ async fn test_quorum_proposal_recv_task() {
             proposals[1].data.clone(),
             leaves[0].clone(),
         )),
+        exact(ViewChange(ViewNumber::new(2))),
```

Comment:
This is a serial output, so this test changed but the liveness test did not need to.